.github/workflows/autocorrect-lint.yml

```diff
@@ -0,0 +1,36 @@
+name: AutoCorrect Lint
+
```

Comment:
Can we add a comment here explaining what autocorrect does (e.g. giving a very short example), link to https://huacnlee.github.io/autocorrect/, and mention that it is only enabled for `zh-CN`?

Reply:
Current only enabled for `zh-CN`, because that we have already fixed the zh-CN contents by a lot of pull request in before.

https://github.com/mdn/translated-content/pulls?q=is%3Apr+is%3Aclosed+author%3Ahuacnlee

But other translations (like `zh-TW`, `jp`) not done, there still have a lot of issues in the contents. We can start on `zh-CN` first and then gradually enable for other translations.

Reply:
I just write a description of this file.